engine/src/hopeit/server/web.py

```diff
@@ -236,6 +240,9 @@ def _setup_app_event_routes(app_engine: AppEngine,
         by same app_engine
     """
     for event_name, event_info in _effective_events(app_engine, plugin).items():
+        if app_engine.app_config.groups and \
```

Comment:
This logic can be simplified for a single group, use a normal if + and statements instead of skipping the unneded ones.